### ISSUE TITLE
Make the share page form more accessible.

### ISSUE THREAD

**Issue by stenington** [mozilla/openbadges-backpack#519]
All controls should have accessible names or labels.

**Comment by mattdigitalme** [comment]
@auralon close?
